### DISCUSSION TITLE
difference between m.draw_features and m.draw_control.data

### DISCUSSION THREAD

**Discussion by suredream** [opengeos/leafmap#682] [Q&A] [answered]
are they same? which one has the latest updates from the controls?

How to keep them sync?

**Comment by giswqs** [comment, marked as answer]
Use m.draw_contorl.data if possible. That's ipyleaflet's built-in attribute. m.drawn_features maybe not be up to date if the features have been edited

**Comment by suredream** [comment]
That's what I observed。 Thank you!
